packages/flutter/lib/src/widgets/icon.dart

```diff
@@ -29,6 +29,8 @@ import 'icon_theme_data.dart';
 /// This example shows how to use [Icon] to create an addition icon, in the
 /// color pink, and 30 x 30 pixels in size.
 ///
+/// ![A pink plus sign](https://flutter.github.io/flutter/assets-for-api-docs/master/assets/widgets/icon.png)
```

Comment:
```suggestion
/// ![A pink plus sign](https://flutter.github.io/assets-for-api-docs/assets/widgets/icon.png)
```
ack sorry. I didn't notice the other segments in the url. This one works. :)

Reply:
Thanks!